core/src/main/kotlin/io/rover/campaigns/core/tracking/AutoScreenTracker.kt

```diff
@@ -48,3 +46,8 @@ internal class AutoScreenTracker : Application.ActivityLifecycleCallbacks {
     override fun onActivityStopped(activity: Activity?) {}
     override fun onActivityCreated(activity: Activity?, savedInstanceState: Bundle?) {}
 }
+
+interface TrackableContentScreen {
+    var contentName: String?
+    var contentID: String?
```

Comment:
I wonder, is it worth also allowing them to override screenName here too? Just in case they have some sort of dynamic activity thingy. or whatnot.

Reply:
I think this helps them break the convention that `screenName` is for a static screen??

Reply:
hmmm well I'll have to do it on iOS since there's no real equivalent to the manifest as far as this goes.

Reply:
also when you trackScreenViewed manually you will have to set it anyway. Maybe we just document it's purpose properly on the field?